main/integration-tests/support/complex/delete-workspace-service-catalog.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-await-in-loop */
+/* eslint-disable no-console */
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const { sleep } = require('@aws-ee/base-services/lib/helpers/utils');
+const _ = require('lodash');
+
+/**
+ * We need a delete method to remove service catalog workspaces
+ * because the deletion done by the environment-sc service is a
```

Comment:
Does environment-sc service delete remove study access on delete?

Reply:
It does not have hard delete anything. It just changes the status of the environment record to "TERMINATED".

I don't need to worry about deleting studies, because I am not using studies in the unit test.